### PR TITLE
Add HLSL to SUPPORTED_LANGUAGES.md

### LINKED ISSUE #2690: [Request] HLSL and ShaderLab support.
It would be nice to have HLSL support in particular.
It would be especially good for people discussing Unity.

Github highlighting supports both of these, maybe you can pull from there?

### PR DESCRIPTION
**Pull request by stef-levesque**
Resolves #2690

### Changes
Add link to [highlightjs-hlsl](/stef-levesque/highlightjs-hlsl)

### CODE REVIEW

**Comment by joshgoebel** [comment]
Awesome.  Please build your grammar inside the HLJS project (`extra` folder, see 3rd party instructions) and then check-in the `dist` folder (which will contain the CDN ready distributable).

File: SUPPORTED_LANGUAGES.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ Languages that listed a **Package** below are 3rd party languages and are not bu
 | Handlebars              | handlebars, hbs, html.hbs, html.handlebars        | |
 | Haskell                 | haskell, hs            |         |
 | Haxe                    | haxe, hx               |         |
+| High-level shader language| hlsl                | [highlightjs-hlsl](https://github.com/stef-levesque/highlightjs-hlsl) |
```
Comment (joshgoebel) [inline SUPPORTED_LANGUAGES.md:93]: Is it more commonly recognized as HLSL to most? We're not afraid of using abbreviations in this column if they are common.

Comment (stef-levesque) [reply SUPPORTED_LANGUAGES.md:93]: Both are used, I'm following the same convention as `OpenGL Shading Language | glsl`

Comment (joshgoebel) [reply SUPPORTED_LANGUAGES.md:93]: OK, we're not 100% consistent here anyways prolly. :-)  Resolving.

**Comment by stef-levesque** [comment]
My bad, the default `.gitignore` removed the `dist/` folder.
Fixed in https://github.com/stef-levesque/highlightjs-hlsl/commit/6ed43745af811680f14f9a9496e39208ce64ff60

**Comment by joshgoebel** [comment]
Please add a CHANGES.md entry also to give your grammar a little more visibility.

**Review by joshgoebel — APPROVED** [review]

File: CHANGES.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 New Languages:
 
 - Added 3rd party RiScript grammar to SUPPORTED_LANGUAGES (#2988) [John C][]
+- Added 3rd party HLSL grammar to SUPPORTED_LANGUAGES (#3002) [Stef Levesque][]
```
Comment (joshgoebel) [inline CHANGES.md:6]: You need to add a link as well around line 13 so it knows where to link you to, etc...

Comment (stef-levesque) [reply CHANGES.md:6]: arg, sry, first contribution :)

File: CHANGES.md [hunk 3]
```diff
@@ -12,6 +13,7 @@ Language grammar improvements:
 - enh(php) Add `mixed` to list of keywords (#2997) [Ayesh][]
 - enh(php) Add support binary, octal, hex and scientific numerals with underscore separator support (#2997) [Ayesh][]
 
+[highlightjs-hlsl]: https://github.com/stef-levesque/highlightjs-hlsl
```
Comment (joshgoebel) [inline CHANGES.md:16]: Typically it's your name, giving you credit (by name)... but I'm ok with this if you'd prefer it... let me know.

File: CHANGES.md (hunk 2 above)
Comment (joshgoebel) [reply CHANGES.md]: No worries.

File: CHANGES.md (hunk 3 above)
Comment (stef-levesque) [reply CHANGES.md:16]: Well I wonder if highlightjs-hlsl could be added to the `highlightjs` organization. Might be easier if other people want to contribute to it.

Comment (joshgoebel) [reply CHANGES.md:16]: https://github.com/highlightjs/highlightjs-hlsl

Comment (stef-levesque) [reply CHANGES.md:16]: Thanks! I'll move the repo and fix the links.

**Review by joshgoebel — APPROVED** [review]

File: CHANGES.md (hunk 3 above)
Comment (stef-levesque) [reply CHANGES.md]: ok, I've moved the repo and fixes the links. Let me know if there's anything missing, or that could be improved 🙂

**Pull request merged**